.github/workflows/build-container.yaml

```diff
@@ -121,7 +121,7 @@ jobs:
     needs: [build-image]
     name: "Publish Image"
     runs-on: ubuntu-latest
-    if: ${{ github.event_name == 'push' && startsWith(github.event.push.ref, 'refs/tags') || (github.event.workflow_dispatch.ref == 'refs/heads/main' && inputs.publish) }}
+    if: ${{ github.event_name == 'push' && startsWith(github.ref, 'refs/tags') || (github.event.name == 'workflow_dispatch' && github.ref == 'refs/heads/main' && inputs.publish) }}
```

Comment:
would we ever create a tag that doesn't represent a release or some other non-publishable event?

Reply:
It's quite possible and maybe even desired